loading.bs

```diff
@@ -331,6 +338,222 @@ Note: Applying the signed exchange's response here has the effect of letting a
 newer HTTP cache entry override a signed exchange's content, and of not storing
 the signed exchange's response in the HTTP cache.
 
+# Subresource substitution # {#subresource-substitution}
+
+To let the UA know that subresources of a page it's prenavigating to, can
+be retrieved as signed exchanges from the source site, this section introduces
+an extension to the HTTP Link header.
+
+* A <dfn dfn-type=dfn>allowed-alt-sxg link</dfn> is a relationship that is used
```

Comment:
Imitate the language used in https://html.spec.whatwg.org/multipage/links.html#link-type-prefetch when defining this new link type. It probably makes sense to describe this as a monkeypatch to HTML so we don't end up with duplication like https://www.w3.org/TR/resource-hints/#prefetch.